### ISSUE TITLE
Time Graph: Support display styles

### ISSUE THREAD

**Issue by tahini** [theia-ide/theia-trace-extension#35]
Some data provider support styles. They should be used when available

**Comment by MatthewKhouzam** [comment]
To be clear, style in this case would be 

- Color
- Size
- Shape (Marker)
- image (one can dream)

Anything else?

**Comment by ebugden** [comment]
⚠️**This issue is related to the Time Graph which has been moved to its own library: [theia-ide/timeline-chart](https://github.com/theia-ide/timeline-chart)**. The code related to this issue can be found in the timeline-chart library and PRs for this issue should be submitted there.

**Comment by mirsky-work** [comment]
Fixed by #344?

**Comment by bhufmann** [comment]
The support for timeline-charts is implemented. We might want to verify if there are still styles that can't be applied by the library. 

For the xy-charts, the styles are not supported yes, afaik.

**Comment by mirsky-work** [comment]
> The support for timeline-charts is implemented. We might want to verify if there are still styles that can't be applied by the library.
> 
> For the xy-charts, the styles are not supported yes, afaik.

Right, so can we close this issue and create a separate one for xy-charts?
